src/app/forms/templates/test-records/section-templates/test/contingency-test-section-group1.template.ts

```diff
@@ -123,7 +123,7 @@ export const ContingencyTestSectionGroup1: FormNode = {
               name: 'prohibitionIssued',
               label: 'Prohibition issued',
               type: FormNodeTypes.CONTROL,
-              value: null,
```

Comment:
The `prohibitionIssued` fields for test sections don't need to be set as false by default since they still show as radiobuttons if it is applicable to the test type. The change in the defects template is needed since it is show as a single check box